### DISCUSSION TITLE
L2 Leafs in L3 Leaf Spine Toplogy

### DISCUSSION THREAD

**Discussion by theotherguy2175** [aristanetworks/avd#3506] [General]
I have basically followed the example of L3 Spine/Leaf in the documentation. Is it possible to actually have some leaves be L2 only in this setup?

**Comment by ClausHolbechArista** [comment]
You would have to define a new type under `node_type_keys` - notice you will replace the defaults when setting this key, so copy the defaults from docs and edit/add as needed. Not all combinations will work well, so but for this case, you should be able to copy l3leaf and set `network_services.l3: false` to create an l2-only VTEP.
